HW1/P2/P2b.py

```diff
@@ -1,3 +1,26 @@
 from P2 import *
 
 # Your code here
+import findspark
+findspark.init('/home/xiaowen/spark')
+import pyspark
+
+# build sparkcontext object
+sc = pyspark.SparkContext(appName="P2")
+
+# initialize 2000*2000 pixels
+i_list = range(2000)
+np.random.shuffle(i_list)  # shuffle i_list
+rdd_1d = sc.parallelize(i_list, 10)
+rdd_2d = rdd_1d.cartesian(rdd_1d)
+
```

Comment:
Instead of randomize one axis and then do the Cartesian, I chose to randomize both axes and then multiply them together. Wondering if that will make the partitioning more random.